diagnostic.md

```diff
@@ -3,36 +3,51 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+- give client-side application a lot of structure right out of the box
+- give app a lot of front-end best practices, standards, and conventions
+- build a state-dependent UI in a proven, standardized way
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+Ember uses a Handlebars templating library. Handlebars templates are just like
+regular HTML or Handlebars, but also give you the ability to embed expressions
+that change what is displayed.
```

Comment:
Yes, ember uses handlebars, with some extras, like the ability to link templates and bind data. 
1/1